types/test.ts

```diff
@@ -0,0 +1,10 @@
+import mdastUtilToString = require('mdast-util-to-string')
+import * as mdastUtilToStringImport from 'mdast-util-to-string'
```

Comment:
If line 2 works and this import can be imported can be used as a function, line 1 works as well. This is testing TypeScript internals.

I’m ok with accepting this, but it’s redundant. I wouldn’t require external contributors to define this test.

Reply:
updated in b132248